package.json

```diff
@@ -9,8 +9,8 @@
   "scripts": {
     "clean": "rm -Rf build node_modules .yarn .eslintcache",
     "prepare": "husky install",
-    "start": "BROWSER=none GENERATE_SOURCEMAP=false react-scripts start",
-    "lint": "eslint --cache . && prettier --write --ignore-unknown .",
+    "start": "BROWSER=none react-scripts start",
+    "lint": "eslint --cache  --fix . && prettier --write --ignore-unknown .",
```

Comment:
Ik denk dat de fix hieruit moet. Die wil je in een aparte lintfix of lint-fx of lint:fix. Zodat je ook de lint kan draaien zonder de code te wijzigen (bijv in de github action)